tests/components/MapboxMap.stories.tsx

```diff
@@ -0,0 +1,88 @@
+import { useCallback } from 'react';
+import { ComponentMeta } from '@storybook/react';
+import { SearchHeadlessContext, Source } from '@yext/search-headless-react';
+
+import { generateMockedHeadless } from '../__fixtures__/search-headless';
+import { MapboxMap, MapboxMapProps } from '../../src/components/MapboxMap';
+import { MapPin } from '../../test-site/src/components/MapPin';
+
+const meta: ComponentMeta<typeof MapboxMap<Location>> = {
```

Comment:
I think this is using some built in Location interface which doesn't seem to be the one we want?